README.md

```diff
@@ -44,15 +44,15 @@ The wic image can also be flashed to the NVME drive using a USB stick with the
 ```
 1. Attach the USB stick, (re)boot the board and select the proper USB drive in
   the UEFI Boot menu.
-2. Select: 'USB Boot (If Drive is present): COM-HPC Yocto Image'
+2. Select: 'USB Boot (If Drive is present): COM-HPC AVA Yocto Image'
```

Comment:
This menu entry should be aligned with the 'wic/ampere-grub.cfg' file

Reply:
Done